### ISSUE TITLE
Unwrap phase node

### ISSUE THREAD

**Issue by abargerbos** [toolsforexperiments/plottr#251]
Would you be open to an unwrap phase button, perhaps similar to the subtract average node/filter/widget (I'm not sure on the naming)? I can imagine the interface should not get too cluttered, but phase wrapping does come up quite frequently and makes it essentially impossible to look at the data. Or would you say the user should take care of this on their data acquisition end?

**Comment by wpfff** [comment]
@abargerbos yes, i think that would be a great addition (even though i do tell my students to always use correct electrical delays in vna measurements, for example ;))
one option would indeed be an additional control like for the average subtraction.
an even better solution would be a more comprehensive filter node that does multiple 'typical' complex data operations (like rotation in iq space, etc).
did you mean to imply that you are willing to give a first prototype a shot? :) 
I'm not yet sure when i myself would have the time to implement this, there's a few projects ahead of this on my end.

**Comment by abargerbos** [comment]
A node like that does sound like it would be very useful! I'm willing to give it a shot (I should be able to get somewhere with the existing code as a template), but I can't yet guarantee when that will be either. To be continued!
